freud/order.pyx

```diff
@@ -531,6 +531,16 @@ cdef class Steinhardt(_PairCompute):
     :math:`q_{lm}` values over all particles before computing the order
     parameter of choice.
 
+    .. note::
+        The value of per-particle order parameter will be set to NaN for
+        particles with no neighbors. We choose this value rather than setting
+        the order parameter to 0 because in more complex order parameter
+        calculations (such as when computing the :math:`w_l`), it is possible
+        to observe a value of 0 for the per-particle order parameter even with
+        a finite number of neighbors. If you would like to ignore this
+        distinction, you can mask the output order parameter values using
+        NumPy: :code:`particle_order[numpy.isnan[particle_order]] = 0`.
```

Comment:
I recommend https://numpy.org/doc/stable/reference/generated/numpy.nan_to_num.html:
```suggestion
        NumPy: :code:`numpy.nan_to_num(particle_order)`.
```

Reply:
@vyasr I don't think you can get infinity here. Regardless, I hope that pointing to the function is probably enough info for users to figure out the rest. I would like to link to the function in NumPy's docs instead of (or in addition to) providing a code snippet.